core/types/src/key_request.rs

```diff
@@ -166,6 +166,26 @@ impl KeyRequestBuilder {
     }
 }
 
+#[derive(Debug, Clone, Hash, PartialEq, Eq)]
+pub struct Key128bit(sgx_key_128bit_t);
+
+new_type_accessors_impls! {
+    Key128bit, sgx_key_128bit_t;
+}
+
+impl Key128bit {
+    /// The underlying size of the key array.
+    // Note: The SGX headers do not have a header for this, they use 16
+    // directly.
+    pub const SIZE: usize = 16;
```

Comment:
Is this constant useful outside this file? Can this be made private?

If it's going to be used outside, we should (ugh) make this into something like a `FixedSize` trait or something...

Reply:
Removed it.  Along with being able to derive `Default` it was over exposing API members thinking users might need to know how to construct a key.  They should get the default and then pass that as a reference to `sgx_get_key()`